packages/features/bookings/lib/handleSeats/types.d.ts

```diff
@@ -2,7 +2,7 @@ import type { Prisma } from "@prisma/client";
 import type z from "zod";
 
 import type { Workflow } from "@calcom/features/ee/workflows/lib/types";
-import type { AppsStatus } from "@calcom/types/Calendar";
+import type { AppsStatus, CalendarEvent } from "@calcom/types/Calendar";
```

Comment:
`CalendarEvent` wasn't imported, so `evt` was treated as `any` instead. Importing it now caused some other type errors that needed to be fixed